src/main/java/org/folio/rest/impl/PurchaseOrderHelper.java

```diff
@@ -652,31 +655,41 @@ private boolean hasNewPoLines(CompositePurchaseOrder compPO, JsonArray poLinesFr
   }
 
   /**
-   * The method compares the permissions that are necessary to manage the acquisition units assignments and the permissions granted by the user,
-   * and if there is no required permission among the granted permissions, it throws {@link HttpException} with status 403.
+   * The method checks if user trying assign order to acquisition unit, if yes that
```

Comment:
Going by what method is doing, think it should be something like `The method checks if the order is assigned to acquisition unit, if yes, then check that if the user has desired permission to assign the record to acquisition unit`

Reply:
Updated